airbyte-cdk/python/unit_tests/sources/file_based/test_scenarios.py

```diff
@@ -127,6 +128,7 @@ def _verify_read_output(output: EntrypointOutput, scenario: TestScenario[Abstrac
 
 def _verify_analytics(analytics: List[AirbyteMessage], expected_analytics: Optional[List[AirbyteAnalyticsTraceMessage]]) -> None:
     if expected_analytics:
+        assert len(analytics) == len(expected_analytics), f"Number of actual analytics messages ({len(analytics)}) did not match expected ({expected_analytics})"
```

Comment:
```suggestion
        assert len(analytics) == len(expected_analytics), f"Number of actual analytics messages ({len(analytics)}) did not match expected ({len(expected_analytics)})"
```

i forgot to add len()

Reply:
i gotchu [77e7567](https://github.com/airbytehq/airbyte/pull/35906/commits/77e756797de61e7e1454c04c2302af5bd9656dd9)